charts/ingress-nginx/values.yaml

```diff
@@ -824,6 +824,7 @@ defaultBackend:
   ##
   nodeSelector:
     kubernetes.io/os: linux
+    openstack-control-plane: enabled
```

Comment:
This will get lost the next time we sync the charts, so what we need to do for this is actually add it into `roles/ingress_nginx/vars/main.yml` into `_ingress_nginx_helm_values`.

Reply:
Thanks for pointing it out.